contents/teams/sales-cs/objectives.mdx

```diff
@@ -1,10 +1,9 @@
-- Hit individual quota targets (Team)
-- Hit and track team revenue goals (Simon)
-- Measure how well we are selling against the vibe-based matrix, then use this for things like lead assignment (Simon)
-- We have messaging for customer data infrastructure documented (Simon)
-- Roll out SafeBase Trust Center and AI security tool (Ben, Sean)
-- Document clear criteria for whether and/or when an account goes to Sales or CS (Simon)
-- Visit all customers spending $100k+ in your book (Team)
-- Make sure we can demo every part of the product. (Sachin, Seb)
-- Documented playbook for cross-sell (Scott + Chris)
-- Try some outbound experiments (Charles, Simon)
+- Hit or exceed individual quota targets (All TAEs)
+- Hit or exceed team quota target (Ben)
+- Generate at least one outbound opportunity each (All TAEs)
+- Close as won at least one outbound opportunity, team-wide (All TAEs)
+- Continue to iterate on outbound motions, especially post-Atlanta (All TAEs)
+- Define and implement sales framework to get people to the best path quickly (Ben + All TAEs)
```

Comment:
> Continue to iterate on outbound motions, especially post-Atlanta (All TAEs)
> Define and implement sales framework to get people to the best path quickly (Ben + All TAEs)

Are these two that different? (We could potentially drop 'continue to iterate' as BAU type work)